proto/protovalidate/buf/validate/validate.proto

```diff
@@ -3278,8 +3278,16 @@ message StringRules {
   // `WellKnown` rules provide advanced constraints against common string
   // patterns
   oneof well_known {
-    // `email` specifies that the field value must be a valid email address
-    // (addr-spec only) as defined by [RFC 5322](https://datatracker.ietf.org/doc/html/rfc5322#section-3.4.1).
+    // `email` specifies that the field value must be a valid email address, for
+    // example "foo@example.com".
+    //
+    // Conforms to the definition for a valid email address from the [HTML standard](https://html.spec.whatwg.org/multipage/input.html#valid-e-mail-address).
+    // Note that this standard willfully deviates from [RFC 5322](https://datatracker.ietf.org/doc/html/rfc5322),
+    // which allows many unexpected forms of email addresses and will easily match
+    // a typographical error. This standard will still match email addresses that
```

Comment:
```suggestion
    // a typographical error. The HTML standard will still match email addresses that
```

Reply:
Feels off to say that the HTML standard matches email addresses, but the current wording isn't very clear either.

Removed the sentence in https://github.com/bufbuild/protovalidate/pull/320/commits/ba24da7b34d646495db4147ebf6d040c3b76dcbd.